packages/hash/frontend/src/components/hooks/useUpdateAuthenticatedUser.ts

```diff
@@ -41,23 +44,27 @@ export const useUpdateAuthenticatedUser = () => {
           return { updatedAuthenticatedUser: authenticatedUser };
         }
 
-        const [
-          {
-            data: { me: latestMeSubgraph },
-          },
-        ] = await refetch();
+        const { data: latestUserEntitySubgraph } = await getEntity({
```

Comment:
Do you need the default depths here? I'm guessing you actually want depths of 0?

Reply:
Whilst testing the signup flow I realised the `getEntity` BP method doesn't work for authenticated users that haven't completed signup. So have ended up replacing this with the `me` query as it was previously in https://github.com/hashintel/hash/pull/1637/commits/a9e2f792e65eccf98954ecef630a89c3b1cfb7bb